website/src/community/in-person.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: section
+title: "Contact Us"
+permalink: /community/in-person/
+section_menu: section-menu/community.html
+redirect_from:
+  - /use/issue-tracking/
+  - /use/mailing-lists/
+  - /get-started/support/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# We are nice, find us in person
+
+While the official communication happens on the mailing list, and you can find us online (see: [contact-us](https://beam.apache.org/community/contact-us/)), we also get together in person at Meetups and Conferences.    
+
+## Meetups
+
+We occassionally meetup in various locations around the globe.  Active or to-be-started meetups include:
```

Comment:
We can included the generic link to find Beam meetups for those not listed here: https://www.meetup.com/topics/apache-beam/

Reply:
@tweise, ah I don't have notifications setup so hadn't seen your comments!  Thanks.  I can update that.